packages/prompts/src/index.ts

```diff
@@ -624,3 +624,41 @@ Only return Node.js code. DO NOT include any English text before or after the No
 
 `;
 };
+
+export const englishStyleGuidePrompt = `# English Style Guide
+
+## Instructions
+
+Your goal is to apply two style guides to any English you find in the code:
+
+1. Google's Developer docs style guide (https://developers.google.com/style)
+2. Dreyer's English
+
+You might see English in places like the following (this is not an exhaustive list):
+
+- Code comments
+- Code strings
+- Any string you find in the code (e.g. the values of properties, variables, etc.)
+- Markdown (headings, text, lists, etc.)
+- HTML attributes, like img alt tags
+- etc.
+
+Please apply the rules of the style guide to ANY English you find.
```

Comment:
```suggestion
Apply the rules of the style guides to any English you find.
```

Reply:
Changed "Please apply the rules of the style guide" to "Apply the rules of the style guides" for consistency and to follow the imperative mood.